### ISSUE TITLE
False positives when using validateAttributeSeparator

### ISSUE THREAD

**Issue by benedfit** [benedfit/jade-lint#16]
The following lines of Jade cause false positives when using validateAttributeSeparator:

```jade
a(style={color: 'red', background: 'green'})
a(class=['red', 'green', 'blue'])
a(class={active: currentUrl === '/'})
```

**Comment by brunowego** [comment]
@benedfit another false positive

```jade
a(href='#') English (US)
```

**Comment by benedfit** [comment]
Thanks @brunowego. Unfortunately this issue won't be solved until I've finished my work on returning the column number of tokens in `jade-lexer`, at which point the regex I currently have can be simplified as I won't need to parse the whole contents of each line of Jade to find pairs of opening and closing attribute brackets

**Comment by gergelykovacs** [comment]
Hi @benedfit 

For extensively used mixins such like e.g.
```jade
+createItem(anItem)(class='aClass anotherClass')
``` 
it is also false positive. I hope you can fix it soon. (I understand it is due to the same as mentioned above.)

Thanks your efforts! Cheers.

**Comment by benedfit** [comment]
Thanks, I'll make sure to include this in the test cases once I've finished the improvements to `jade-lexer`

**Comment by benedfit** [comment]
I'm back on the case with the linter and will hopefully have an update on this soon

**Comment by benedfit** [comment]
Hi both, these issues have been resolved in v2.0.0

**Comment by brunowego** [comment]
Thanks @benedfit
